apps/crn-frontend/src/network/users/UserProfile.tsx

```diff
@@ -64,6 +59,23 @@ const User: FC<Record<string, never>> = () => {
 
   const isOwnProfile = currentUser?.id === user?.id;
 
+  const [currentTime] = useState(startOfMinute(new Date()));
+
+  const upcomingEventsResult = useEvents(
+    getEventListOptions(
+      currentTime,
+      false,
+      {
+        searchQuery: '',
+        currentPage: 0,
+        pageSize: 1,
```

Comment:
Is it not possible to have parameters match what we do for upcoming events without a search query? It would save us an algolia query in the most common case.